### ISSUE TITLE
Placed object is not updated when source image file is changed

### ISSUE THREAD

**Issue by UliAbo** [bjorn/tiled#2810]
**Current Behavior:** 
When the image file of an Object is changed (file name is still the same), already placed Objects are not updated. The User has to remove the Object first and add it again with the updated image file.

**New Behavior:**
When the image file of an Object is changed (file name is still the same), already placed Objects are updated automatically (like it's already the case with updated image files for Tilesets).

**Comment by UliAbo** [comment]
Hmm, I think this is already closed? Due to the Reload function?

Or what about an automatic reload function, which could be enabled? This would also be nice to have not only for the TMX, but also TSX files.

**Comment by bjorn** [comment]
Yes, automatic reloading of images has only been implemented for tilesets based on a tileset image, not for image collection tilesets, nor for image layers (#825). It would be nice to find a more generic solution for this, so let's keep this issue open.

In the meantime, indeed the manual Reload Tilesets action (Ctrl+T) can be used as a workaround.

**Comment by eishiya** [comment]
Related? issue: Image Layers are not updated when the source images/maps change either, and there's doesn't appear to be anything like Ctrl+T for those. Ctrl+R can reload the whole map, but that's not always appropriate because the map being worked on may not be saved.
